### ISSUE TITLE
Idea for new project... simple DD1 Floppy controller......

### ISSUE THREAD

**Issue by pocoaust** [revaldinho/cpc_ram_expansion#12]
Hi mate,

Love your work, have built the backplane and ROM board.  No issues just an idea

Have you considered a simple DD1 Floppy interface replacement.  They are getting very hard to find. The rom could go on the ROM board, and a small addon board could carry the fdd controller and support logic on your bus expansion. 

I have attached the DDI schematic, could you give me your thoughts on this.

Best Regards
Paul

![CPC664_Disk_Interface_Schematic](https://user-images.githubusercontent.com/89445361/203709381-a2d3d382-2f55-4eab-8bb2-9d8095465225.png)

**Comment by revaldinho** [comment]
Thanks for your comments.

It's a good idea. I mean, there are plenty of SD card alternatives to the DDI-1 around for the 464/664 but I think that only Zaxon's DDI-3/4/5 boards provide physical drive interfaces.  I actually do have a DDI-3 and it works very well, also with a Gotek as Drive B on the expansion connector.

Deleting the ROM and control circuitry is definitely the thing to do. I would like to keep any board smaller than 80x100mm to fit the freeware- Eagle license (100x100mm is also the limit for the cheap proto-packs from Seed). This would probably fit without the ROM. 

I'll think about it. I have a couple of other projects in progress at the moment, so it may take a while to get to this and although I don't really need one myself,  I agree that it would be worth making a clone more generally available.

**Comment by pocoaust** [comment]
That's great.  I am just a novice teaching myself in retirement, but am
happy to pay for test boards and have plenty of 765 ic etc which I could
send you.  Anyway let me know if there is anything I can do on my side and
please keep me in the loop.

Thanks Paul in australia.
Take care...

**Comment by pocoaust** [comment]
Just an after thought, 3.5" style connector so you could support gotek as
mentioned would be ideal.

Take care. Paul;-)

**Comment by pocoaust** [comment]
Tried to send this to you but github said was 2 large.

Here is link....

https://www.grimware.org/lib/exe/fetch.php/documentations/hardware/amstrad.cpc/manuals/service.manual.ddi-1.fd-1.pdf

Let me know if You need anything else.

Take care
Paul

**Comment by grafzahl12345** [comment]
> 

Hi revald, 
are you also still useing eagle and is there enough space in the cpld for a second adress select logic? So i'm intrested too, you can cut me in if you like.

**Comment by revaldinho** [comment]
Yes, for these projects at least I'm still using the netlister.py/freeware Eagle PCB tools.

For a DDI-1 clone without a ROM and the associated decoding I don't think there will be any need for a CPLD.  It may be possible to reduce the chip count by refactoring what's left with the ROM logic gone, so I think that it should all fit the in the 80x100mm constraint without any need to go for a CPLD.
